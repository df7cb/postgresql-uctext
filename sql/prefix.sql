select * from uc where u =|| 'df';
select * from uc where u =|| 'DF';
select * from uc where t =|| 'df';
select * from uc where t =|| 'DF';
