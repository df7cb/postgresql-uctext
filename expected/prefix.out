select * from uc where u =|| 'df';
   u   |   t   
-------+-------
 DF7CB | df7cb
 DF7CB | df7cb
(2 rows)

select * from uc where u =|| 'DF';
   u   |   t   
-------+-------
 DF7CB | df7cb
 DF7CB | df7cb
(2 rows)

select * from uc where t =|| 'df';
   u   |   t   
-------+-------
 DF7CB | df7cb
 DF7CB | df7cb
(2 rows)

select * from uc where t =|| 'DF';
   u   |   t   
-------+-------
 DF7CB | df7cb
 DF7CB | df7cb
(2 rows)

